### DISCUSSION TITLE
Message on all pages

### DISCUSSION THREAD

**Discussion by alainrobert** [requarks/wiki#5955] [Help / Questions] [answered]
Using the latest version of 2.5, I am looking for a way to have a message popup  or displayed on every page to notify users of changes. 
Looking for something similar to the donate popup and messages that appears sometimes on wikipedia for example.   

Is this something already possible?

**Comment by fnafox** [comment, marked as answer]
You could accomplish this fairly easily with code injection.
`Admin > Theme > Code Injection`

Example below is for cookie consent but the idea is the same:
https://github.com/requarks/wiki/discussions/3993#discussioncomment-4370705

**Reply by alainrobert** [reply]
thanks, I had missed the code injection section
